package.json

```diff
@@ -9,11 +9,17 @@
     "url": "git@github.com:resin-io/dockerfile-template"
   },
   "author": "Aleksis Brezas <abresas@resin.io>",
+  "contributors": [
+    {
+      "name": "Cameron Diver",
+      "email": "cameron@resin.io"
+    }
+  ],
   "license": "Apache-2.0",
   "scripts": {
-    "pretest": "coffeelint lib.coffee dockerfile-template.coffee ./test",
+    "pretest": "resin-lint lib.coffee dockerfile-template.coffee ./test",
     "test": "mocha --compilers coffee:coffee-script/register",
-    "prepublish": "gulp && (echo '#!/usr/bin/env node' | cat - dockerfile-template.js > dockerfile-template)"
+    "prepublish": "gulp"
```

Comment:
And this just reminded me of the whole npm4 prepublish only thing: https://resin.io/blog/safely-migrating-away-from-prepublish-with-npm-4/
Copying https://github.com/resin-io/typed-error/commit/2d7b9c9c47ba32231d5ed2d2d624c77d6783905e is probably the easiest way to set it up